stable/mariadb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 0.3.0
+version: 0.3.1
```

Comment:
It would be good to discuss how we want to version charts at some point, some general guidelines would be good.